README.md

```diff
@@ -133,6 +126,106 @@ jobs:
           atmos-gitops-config-path: ./.github/config/atmos-gitops.yaml    
 ```
 
+
+### Migrating from `v1` to `v2`
+
+`v2` works only with `atmos >= 1.63.0`
+
+`v2` drops `install-terraform` input because terraform is not required for affected stacks call
+
+`v2` drop `atmos-gitops-config-path` input and  `./.github/config/atmos-gitops.yaml`  config. Now you have to use GitHub actions inputs and `atmos.yaml` to specify that configuration.
+
+The following configuration fields now moved to GitHub action inputs with the same names
```

Comment:
```suggestion
The following configuration fields moved to the GitHub Action inputs with the same names. Alternatively, you can set `ATMOS_VERSION` or `ATMOS_CONFIG_PATH` environment variables.
```

Reply:
That's the wrong statement. GHA will override `ATMOS_CONFIG_PATH ` env var.